app/stores/toplevel/WalletStore.js

```diff
@@ -221,10 +221,8 @@ export default class WalletStore extends Store {
     );
   }
 
-  @computed get first(): null | PublicDeriver<> {
-    return this.publicDerivers.length === 0
-      ? null
-      : this.publicDerivers[0];
+  @computed get allWallets(): Array<PublicDeriver<>> {
```

Comment:
We don't need a computed function for this. You can just use `publicDerivers` directly (it's a observable property so it will handle updates. Computed functions are useful when you need to do some calculation involving observable variables to get a final result)